### PR TITLE
Copy video title

### LINKED ISSUE #3768: Copy the media title
#### Describe the feature you want

I want copy the media title (example: long click the title text to copy)
#### Is your feature request related to a problem? Please describe it

No.
#### Additional context

Sorry for my Engrish :(
#### How will you/everyone benefit from this feature?

We will be able to copy the title and share it.

### PR DESCRIPTION
**Pull request by adinilfeld**
#### What is it?
- [ ] Bug fix (user facing)
- [x] Feature (user facing)
- [ ] Code base improvement (dev facing)
- [ ] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Allows user to long-press on the video title in the detail screen to copy it to the clipboard
- Brief "Copied to clipboard" toast appears

#### Fixes the following issue(s)

closes #3768 

#### Testing apk

[app-debug.apk.zip](https://github.com/TeamNewPipe/NewPipe/files/4761426/app-debug.apk.zip)

#### Agreement
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java [hunk 1]
```diff
@@ -583,6 +585,9 @@ protected void initViews(final View rootView, final Bundle savedInstanceState) {
     protected void initListeners() {
         super.initListeners();
 
+        videoTitleRoot.setLongClickable(true);
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java:588]: I don't think this line is needed, as `setOnLongClickListener` already makes the view long clickable: https://developer.android.com/reference/android/view/View#setOnLongClickListener(android.view.View.OnLongClickListener)

File: app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java [hunk 2]
```diff
@@ -130,11 +127,7 @@ public void updateFromItem(final InfoItem infoItem,
...
             if (!AndroidTvUtils.isTv(itemBuilder.getContext())) {
-                ClipboardManager clipboardManager = (ClipboardManager) itemBuilder.getContext()
-                        .getSystemService(Context.CLIPBOARD_SERVICE);
-                clipboardManager.setPrimaryClip(ClipData.newPlainText(null, commentText));
-                Toast.makeText(itemBuilder.getContext(), R.string.msg_copied, Toast.LENGTH_SHORT)
-                        .show();
+                ShareUtils.copyToClipboard(itemBuilder.getContext(), commentText);
             } else {
                 openCommentAuthor(item);
             }
```
Comment (Stypox) [inline app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java:133]: Even though this is unrelated from your changes, could you invert the two branches of the if, while also removing the `!` ?
```suggestion
            if (AndroidTvUtils.isTv(itemBuilder.getContext())) {
                openCommentAuthor(item);
            } else {
                ShareUtils.copyToClipboard(itemBuilder.getContext(), commentText);
            }
```

File: app/src/main/java/org/schabi/newpipe/util/ShareUtils.java [hunk 3]
```diff
@@ -78,4 +81,27 @@ public static void shareUrl(final Context context, final String subject, final S
...
+
+    /**
+     * Copy the text to clipboard, and indicate to the user whether the operation was completed
+     * successfully using a Toast.
+     *
+     * @param context the context to use
+     * @param text    the text to copy
+     */
+    public static void copyToClipboard(final Context context, final String text) {
+        ClipboardManager clipboardManager =
```
Comment (wb9688) [inline app/src/main/java/org/schabi/newpipe/util/ShareUtils.java:93]: This variable could be `final`

File: app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java (hunk 2 above)
Comment (wb9688) [reply app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java]: I made that code, the logic behind it was that non-TV is more common
